hasher-matcher-actioner/.devcontainer/post-create

```diff
@@ -14,4 +14,6 @@ python3 -m pip install --upgrade pip
 python3 -m pip install -r requirements-dev.txt
 
 # Install javascript toolchain
-cd webapp && npm install && cd ..
+cd webapp && npm install && npm install react-waypoint --save && yarn add react-waypoint
```

Comment:
if you updated `package.json` this shouldn't be needed

Reply:
It doesn't look like package.json is a tracked file in git, so is it somehow created in devcontainer creation and will find this package?